src/applications/claims-status/tests/e2e/08.claim-letters-keyboard-only.cypress.spec.js

```diff
@@ -1,14 +1,19 @@
-import featureToggleEnabled from './fixtures/mocks/claim-letters/feature-toggle-enabled.json';
 import claimLetters from './fixtures/mocks/claim-letters/list.json';
 
 describe('Claim Letters Page', () => {
   beforeEach(() => {
     cy.intercept('GET', '/v0/claim_letters', claimLetters.data).as(
       'claimLetters',
     );
-    cy.intercept('GET', '/v0/feature_toggles?*', featureToggleEnabled).as(
-      'featureToggleEnabled',
-    );
+
+    cy.intercept('GET', '/v0/feature_toggles*', {
```

Comment:
I'm not totally sure why this is needed, but this code is in the other claim letters test file so I added it here